EIPS/eip-xxxx.md

```diff
@@ -0,0 +1,72 @@
+---
+
+eip: xxxx  
```

Comment:
```suggestion
eip: 7790 
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.